cli/index.ts

```diff
@@ -86,48 +124,26 @@ createYargsCli({
       format: "compile [project-dir]",
       description:
         "Compile the dataform project. Produces JSON output describing the non-executable graph.",
-      positionalOptions: [
-        {
-          name: "project-dir",
-          option: {
-            describe: "The directory of the Dataform project.",
-            default: "."
-          }
-        }
-      ],
+      positionalOptions: [projectDirOption],
       options: [
-        {
-          name: "default-schema",
-          option: {
-            describe: "An optional default schema name override"
-          }
-        },
-        {
-          name: "assertion-schema",
-          option: {
-            describe: "An optional assertion schema name override"
-          }
-        },
+        defaultSchemaOption,
+        assertionSchemaOption,
         {
           name: "watch",
           option: {
-            describe: "Watch the changes in the project directory.",
+            describe: "Whether to watch the changes in the project directory.",
             type: "boolean",
             default: false
           }
         }
       ],
       processFn: argv => {
-        const projectDir = path.resolve(argv["project-dir"]);
-        const defaultSchemaOverride = !!argv["default-schema-override"]
-          ? path.resolve(argv["default-schema-override"])
-          : "";
-        const assertionSchemaOverride = !!argv["assertion-schema-override"]
-          ? path.resolve(argv["assertion-schema-override"])
-          : "";
+        const projectDir = argv["project-dir"];
+        const defaultSchemaOverride = argv["default-schema"];
+        const assertionSchemaOverride = argv["assertion-schema"];
 
         compileProject(projectDir, defaultSchemaOverride, assertionSchemaOverride).then(() => {
-          if (argv.watch) {
+          if (argv["watch"]) {
```

Comment:
What's the difference?

Reply:
Oh there is none, I'm just trying to enforce some consistency when accessing args.